wolfcrypt/src/ecc.c

```diff
@@ -5165,7 +5165,7 @@ int wc_ecc_sign_hash_ex(const byte* in, word32 inlen, WC_RNG* rng,
    mp_int  e_lcl;
 #endif
 
-#if defined(WOLFSSL_ECDSA_SET_K) || \
+#if defined(WOLFSSL_ECDSA_SET_K) || defined(WOLFSSL_ECDSA_SET_K_ONE_LOOP) || \
```

Comment:
Where is this define documented in the code?

Reply:
Comment added at top.